src/main/java/io/appium/java_client/HasSessionDetails.java

```diff
@@ -34,26 +36,47 @@
     @SuppressWarnings("unchecked")
     default Map<String, Object> getSessionDetails() {
         Response response = execute(GET_SESSION);
+        Map<String, Object> resultMap = Map.class.cast(response.getValue());
+
+        //this filtering was added to clear returned result.
```

Comment:
-> was added to avoid unexpected NPE on _putAll_ call if the _response_ map contains null value(s)

Reply:
@mykola-mokhnach Not only for this purpose. I think empty strings is not useful data and it increases complexity of further operations. So let there will be something useful or null-value.

Reply:
of course, just mention the same thing in the comment